salt/fileclient.py

```diff
@@ -642,21 +643,52 @@ def swift_opt(key, default):
 
         destfp = None
         try:
+            # Tornado calls streaming_callback on redirect response bodies.
+            # But we need streaming to support fetching large files (> RAM avail).
+            # Here we working this around by disabling recording the body for redirections.
+            # The issue is fixed in Tornado 4.3.0 so on_header callback could be removed
+            # when we'll deprecate Tornado<4.3.0.
+            # See #27093 and #30431 for details.
+            write_body = [False]
```

Comment:
Why is this a list and not simply a bool? I imagine its some scoping thing, but if there is a good reason I'd put a comment-- only because someone will have the same question and "clean it up".